src/Tizen.WebView/Tizen.WebView/PolicyDecisionEventArgs.cs

```diff
@@ -0,0 +1,236 @@
+        public void Use()
+        {
+            Interop.ChromiumEwk.ewk_policy_decision_use(_handle);
+        }
+
+        /// <summary>
+        /// Checks if frame requested in policy decision is main frame.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsMainFrame()
+        {
+            return Interop.ChromiumEwk.ewk_policy_decision_is_main_frame(_handle);
+        } 
+    }
+
+    /// <summary>
+    /// This class is derived from PolicyDecisionEventArgs.
+    /// This class provides the properties for New Window Policy of WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class NewWindowPolicyEventArgs : PolicyDecisionEventArgs 
+    {
+        private IntPtr _handle;
+        internal NewWindowPolicyEventArgs(IntPtr handle) : base(handle)
+        {
+            _handle = handle;
+        }
+        
+        internal static NewWindowPolicyEventArgs CreateFromSmartEvent(IntPtr data, IntPtr obj, IntPtr info)
+        {
+            return new NewWindowPolicyEventArgs(info);
+        }
+    }
+
+
+    /// <summary>
+    /// This class is derived from PolicyDecisionEventArgs.
+    /// This class provides the properties for Navigation Policy of WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class NavigationPolicyEventArgs : PolicyDecisionEventArgs
+    {
+        private IntPtr _handle;
+        internal NavigationPolicyEventArgs(IntPtr handle) : base(handle)
+        {
+            _handle = handle;
+        }
+
+        /// <summary>
+        /// Gets the Navigation Type of policy.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public NavigationType NavigationType
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_navigation_type_get(_handle);
+	    }
+        }
+
+        /// <summary>
+        /// Gets the cookie.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Cookie
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_cookie_get(_handle);
+            }
+        }
+
+        internal static NavigationPolicyEventArgs CreateFromSmartEvent(IntPtr data, IntPtr obj, IntPtr info)
+        {
+            return new NavigationPolicyEventArgs(info);
+        }
+    }
+
+    /// <summary>
+    /// This class is derived from PolicyDecisionEventArgs.
+    /// This class provides the properties for Response Policy of WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ResponsePolicyEventArgs : PolicyDecisionEventArgs
+    {
+        private IntPtr _handle;
+        internal ResponsePolicyEventArgs(IntPtr handle) : base(handle)
+        {
+            _handle = handle;
+        }
+
+        /// <summary>
+        /// Gets response status code.
```

Comment:
Could explain about response status code?
Is it means a http renpose code?
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

Reply:
Yes, it is.